in-depth/submission_guidelines.md

```diff
@@ -86,6 +86,10 @@ Heading
 
 : The name of your theme or plugin.
 
+View
+
+: Choose between the image oriented `jumbovision` layout, or the text-oriented `card` layout.  The default for Themes is `jumbovision`, while plugins use `card`.  This option only needs to be specified if you'd like to use the opposite layout.
```

Comment:
Would we really want people changing the button text though?  It just seems kind of inconsistent if we let people *choose* what they'd like it to say.

On that note, one of my very first To-Do items :point_up: was to decide what that button should say. (Download, View on GitHub, More Info, etc).